src/libprojectM/MilkdropPresetFactory/Parser.hpp

```diff
@@ -29,7 +29,7 @@
 #ifndef _PARSER_H
 #define _PARSER_H
 //#define PARSE_DEBUG 2
-#define PARSE_DEBUG 0
+#define PARSE_DEBUG 2
```

Comment:
want to commit this?

Reply:
Ah, good catch! This should rather be done as a CMake flag, and/or only be enabled in debug builds.